boards/nxp/frdm_imx93/frdm_imx93_mimx9352_a55.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx93_a55.dtsi>
+#include "frdm_imx93-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
```

Comment:
keep alphabetical order please with "frdm_imx93-pinctrl.dtsi" at the end

Reply:
I think it's better to group the includes as "dtsi" and "headers". If we have multiple headers we can keep alphabetical order within its category.

Reply:
won't fix, as explained above

Reply:
yes, fully agreed that it's just fine as-is, and how it's done in most boards (including NXP :) )